src/js/models/menubar.js

```diff
@@ -466,6 +467,12 @@ define(function (require, exports, module) {
             "TOGGLE_SINGLE_COLUMN_MODE": { "checked": (preferences.get("singleColumnModeEnabled", false) ? 1 : 0) }
         });
 
+        if (preferences.get("singleColumnModeEnabled")) {
+            headlights.logEvent("user-interface", "panels", "single-column-mode-enabled");
+        } else {
+            headlights.logEvent("user-interface", "panels", "single-column-mode-disabled");
+        }
+
```

Comment:
These events are logged every time the menu mode updated, maybe consider moving these to the button components?